src/reviewerQueue.js

```diff
@@ -6,6 +6,8 @@
 //   HUBOT_GITHUB_ORG (required)
 //   HUBOT_GITHUB_REVIEWER_TEAM (required)
 //     github team id. this script picks the next eligible reviewer off a queue
+//   HUBOT_REVIEWER_SHADOWS (optional)
+//     map of reviewer github logins to reviewer shadow logins
```

Comment:
Can you describe what this data is supposed to look like and how it's formatted? These comments wind up as help strings in Hubot.

Reply:
Done.